pallets/loans-ref/src/lib.rs

```diff
@@ -36,9 +36,14 @@
 //! The whole pallet is optimized for the more expensive extrinsic that is
 //! [`Pallet::update_portfolio_valuation()`] that should go through all active loans.
 
-pub mod migrations;
+pub mod migrations {
+	pub mod nuke;
+	pub mod v1;
+}
+
```

Comment:
We should not keep them around forever or hide them behind a feature flag to not bloat the wasm. But rather minor.